DependencyInjection/HttplugExtension.php

```diff
@@ -297,6 +298,7 @@ private function configureClient(ContainerBuilder $container, $clientName, array
 
         $container
             ->register($serviceId, PluginClient::class)
+            ->setPublic(true)
```

Comment:
How can we avoid to set this one public just for tests?

Reply:
Psudo code:
```
$services = $container->getAllServiceDefinitionKeys()
foreach ($services as $service) {
  if ($service->getId() matches "httplug.client.*") {
    $container->getDefinition($service)->setPublic(true);
  }
}
```

I do not know any better... sorry

Reply:
Btw, will the tests fail if we remove this line?

Reply:
if we keep the compiler pass, this line can be reverted, right?